### ISSUE TITLE
Starting mosh at macOS system startup problems: mosh-server needs a UTF-8 native locale to run

### ISSUE THREAD

**Issue by ylluminate** [mobile-shell/mosh#927]
I'm running `mosh-server` on a macOS server. I can start it up just fine via an interactive shell with `mosh-server` and I can connect to it relatively fine via `mosh --server=/usr/local/bin/mosh-server username@localhost` (not sure why I'm having to pass the server path...), but the real issue is when I try to fire up `mosh-server` in `Lingon X`.

I get the following output when I run a test run of the command `/usr/local/bin/mosh-server`:
```
1: Operation not permitted

mosh-server needs a UTF-8 native locale to run.

Unfortunately, the local environment ([no charset variables]) specifies
the character set "US-ASCII",

The client-supplied environment ([no charset variables]) specifies
the character set "US-ASCII".

LANG=
LC_COLLATE="C"
LC_CTYPE="C"
LC_MESSAGES="C"
LC_MONETARY="C"
LC_NUMERIC="C"
LC_TIME="C"
LC_ALL=
```
Do I need to set some environment variables in some fashion in the startup command?

**Comment by keithw** [comment]
You do not need to run `mosh-server` yourself unless you are trying to debug mosh. Please see the usage instructions at https://mosh.org or in the `mosh(1)` man page. Generally speaking, simply running the `mosh` command at the client is enough. The `mosh` script will remotely execute a `mosh-server` on the server to handle your connection and generate a session, and then execute `mosh-client` locally to connect to the server.

**Comment by ylluminate** [comment]
Gotcha, thanks.

**Comment by jessebett** [comment]
Not sure why this closed solution was accepted. The issue has something to do with UTF-8 vs US-ASCII and @keithw comment did not give any information to resolve this.https://github.com/mobile-shell/mosh/issues/1020

**Comment by keithw** [comment]
This user was running mosh-server directly, which is generally only done when debugging (it's not the normal way to use Mosh). If you are having a problem running the "mosh" command (as explained on the Mosh website) to connect to another computer, please paste the full error message you are getting and we can try to help you fix your setup. Here is a user-support issue of people who have had similar trouble (and you can see how we helped them fix it): https://github.com/mobile-shell/mosh/issues/793

**Comment by anothersailor** [comment]
Well, I got there because my iPad mosh does not find mosh-server on my Mac when I try to connect with mosh (using blink). I checked on the Mac (os Monterey 12.3), the mosh server is found by the shell (zsh) when I type "which mosh-server". So I tried to run it from the Mac, and this is where I got the error above.
If this is not the problem, then what could be the reason for the IPAD not finding the server with SSH ?
Thanks,
E.
